processor/PlanExecutor.go

```diff
@@ -235,10 +263,10 @@ func CreatePlanFromURLList(config *config.Configuration) core.Plan {
 		}
 		action := http.CreateAction()
 
-		action.URL = request.URL.String()
-		action.Method = request.Method
-		action.Headers = request.Header
-		action.Body = body
+		action.State.URL = request.URL.String()
```

Comment:
It could be but it would have to go back into a string form as we use it in 

```go
http.NewRequest(...
```

which needs URL as a string.  I would opt for another ticket to revise this as an enhancement.